src/NzbDrone.Core.Test/ParserTests/LanguageParserFixture.cs

```diff
@@ -298,6 +298,8 @@ public void should_parse_language_arabic(string postTitle)
         [TestCase("Pulp Fiction.eng.sub")]
         [TestCase("Pulp.Fiction.eng.forced.sub")]
         [TestCase("Pulp-Fiction-eng-forced.sub")]
+        [TestCase("2_Eng.srt")]
+        [TestCase("2_English.srt")]
```

Comment:
Test cases:
2_Eng -- old subtitle filename format
2_English -- newer subtitle filename format

Reply:
Perhaps add some test cases for `3_Eng.srt`, `4_English.srt`, etc.